src/js/initializers/initStore.js

```diff
@@ -27,7 +28,10 @@ export default async () => {
     initialState,
     composeWithDevTools(
       applyMiddleware(
-        reduxThunk.withExtraArgument({zealot: createZealot("localhost:9867")})
+        reduxThunk.withExtraArgument({
+          zealot: createZealot("localhost:9867"),
+          globalDispatch
```

Comment:
Are you thinking we should be limiting the scope of `globalDispatch` to be less global with this addition?